pkg/services/libraryelements/database.go

```diff
@@ -882,3 +888,43 @@ func contains(slice []string, element string) bool {
 	}
 	return false
 }
+
+func getFoldersWithMatchingTitles(c context.Context, l *LibraryElementService, signedInUser identity.Requester, query model.SearchLibraryElementsQuery) ([]string, error) {
+	var foldersWithMatchingTitles []string
+	if len(strings.TrimSpace(query.SearchString)) > 0 {
```

Comment:
nit: In Go we try to keep the code aligned to the left. To do so, we prefer to have more than one return statement and return as soon as possible. In this case, for example, it would be better this approach to remove the nested if/else condition.
```suggestion
	if len(strings.TrimSpace(query.SearchString)) <= 0 {
		return nil, nil
	}
```